packages/node-core/src/indexer/fetch.service.spec.ts

```diff
@@ -372,8 +401,81 @@ describe('Fetch Service', () => {
     expect(enqueueBlocksSpy).toHaveBeenCalledWith([2, 3, 4, 6, 8, 9, 10, 12, 15, 18], 18);
   });
 
+  it('can check should useModuloHandlersOnly', () => {
+    dataSources = [
+      {
+        kind: 'mock/DataSource',
+        startBlock: 1,
+        mapping: {
+          file: '',
+          handlers: [
+            {
+              kind: 'mock/Handler',
+              handler: 'mockFunction',
+              filter: {
+                modulo: 150,
+              },
+            },
+          ],
+        },
+      },
+    ];
+    // 1 modulo handler only
+    expect((fetchService as any).useModuloHandlersOnly(dataSources)).toBeTruthy();
+    dataSources[0].mapping.handlers.push({
+      kind: 'mock/Handler',
+      handler: 'mockFunction',
+      filter: {},
+    });
+    // 2 handlers, only 1 has modulo filter
+    expect((fetchService as any).useModuloHandlersOnly(dataSources)).toBeFalsy();
+  });
+
+  it('can exclude modulo blocks', () => {
```

Comment:
Theres too much use of `any` in these tests. It makes them very fragile and to me is a sign the code is not easy to work with. 

We have a TestFetchService which can be extended to make setting test data easier, but calling functions with any and validating the result is poor practice and makes it very hard to make any changes to fetch service because we have no type checking and private functions shouldn't need to be concerned with any tests.